app-electronics/nextpnr/autobuild/defines

```diff
@@ -8,6 +8,9 @@ PKGDES="A portable FPGA place and route tool"
 CMAKE_AFTER="-DARCH=all+alpha \
              -DMISTRAL_ROOT=$SRCDIR/../mistral \
              -DUSE_OPENMP=ON \
-             -DBUILD_GUI=ON"
+             -DBUILD_GUI=ON \
+             -DHIMBAECHEL_GOWIN_DEVICES=all \
+             -DHIMBAECHEL_PRJXRAY_DB=$SRCDIR/../prjxray-db \
+             -DHIMBAECHEL_XILINX_DEVICES=xc7s50;xc7a50t;xc7a100t;xc7a200t;xc7k70t;xc7z010;xc7z020"
```

Comment:
Might need a comment explaining this line (or is there an option to enable all supported devices?).

Reply:
Nope, these devices are just what prjxray-db supports now.

Reply:
> Nope, these devices are just what prjxray-db supports now.

So maybe add some instructions on how to check for newly supported devices in future updates?